packages/jest-environment-enzyme/README.md

```diff
@@ -11,7 +11,7 @@ This package will also simplify your test files by declaring React, and enzyme w
 The setup can be as simple as this:
 
 ```
-yarn add jest-environment-enzyme jest-enzyme enzyme-adapter-* --dev
+yarn add jest-environment-enzyme jest-enzyme enzyme-adapter-react-* --dev
```

Comment:
I actually intentionally left this off. One of the reasons enzyme switched to adapters was to support other react-like libraries (preact, inferno, etc). So I avoided stating react incase someone came here wanting to use this library for one of those other libs. 

so if you remove this I'll gladly take the json fix!